internal/gitea/gitea.go

```diff
@@ -0,0 +1,14 @@
+package gitea
```

Comment:
It's good practice to add a package comment that will appear at the top of the docs (note how the [docs for the github package](https://pkg.go.dev/github.com/jnsgruk/releasegen@v0.4.3/internal/github) for example doesn't have one).

Typically they're one-liners starting with "Package foo ...", but they can be [long too](https://cs.opensource.google/go/go/+/refs/tags/go1.21.5:src/time/time.go;l=5). Something like:

```
// Package gitea implements support for Gitea repositories using the Gitea API client.
```